tests/functional/base_test.go

```diff
@@ -70,6 +70,9 @@ func GetDefaultDNSMasqSpec() map[string]interface{} {
 			Key:    "server",
 			Values: []string{"1.1.1.1"},
 		},
+    {
```

Comment:
Indentation feels off at least on the github interface.

Reply:
not only that, but I missed the closing bracket...